src/testRunner/unittests/services/hostNewLineSupport.ts

```diff
@@ -12,7 +12,7 @@ namespace ts {
                 getCompilationSettings: () => settings,
                 getScriptFileNames: () => map(files, f => f.unitName),
                 getScriptVersion: () => "1",
-                getScriptSnapshot: name => snapFor(name),
+                getScriptSnapshot: (name: string) => snapFor(name),

```

Comment:
Is this change needed?

Reply:
Not anymore. It was when there was an internal overload.